app/src/main/java/vn/mbm/phimp/me/leafpic/data/providers/MediaStoreProvider.java

```diff
@@ -130,6 +130,7 @@ private static int getAlbumCount(Context context, long id) {
 										   + MediaStore.Files.FileColumns.MEDIA_TYPE + "=? ) and " + MediaStore.Files.FileColumns.PARENT + "=?";
 
 		String[] selectionArgs = new String[] {
+						String.valueOf(MediaStore.Files.FileColumns.MEDIA_TYPE),
```

Comment:
@vinaysajjanapu I don't know how the query table is working but I observed that the 1st index of selectionArgs is accounting for the media count, so adding the 0th index as media_type makes the second index as media_type_image which fixes the issue.